cfgov/scripts/enforcement_transfer.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import unicode_literals
+
+from v1.models import (
+    BrowseFilterablePage, CFGOVPageCategory, DocumentDetailPage
+)
+from v1.models.learn_page import EnforcementActionPage, EnforcementActionStatus
+from v1.util.migrations import get_stream_data, set_stream_data, strip_tags
+
+
+ENFORCEMENT_PARENT_ID = 1327
+
+STATUS_OPTIONS = [
+    'Post Order/Post Judgment',
+    'Expired/Terminated/Dismissed',
+    'Pending Litigation'
+]
+
+
+def clean_status(s):
+    return s.replace('-', ' ').replace('udge', 'udg').title()
+
+
+def get_metadata_value(item, variable):
+    return strip_tags(item['value']['blob']) or None
+
+
+def transfer():
+    enforcement_actions = BrowseFilterablePage.objects.get(
+        pk=ENFORCEMENT_PARENT_ID
+    )
+
+    errors = []
+    for page in DocumentDetailPage.objects.all():
+        url = page.get_url()
+
+        if not url or 'policy-compliance/enforcement/actions' not in url:
+            continue
+        if 'enforcement-action-definitions' in url:
+            continue
+
+        keys = vars(page)
+
+        old_categories = page.categories.all()
+        if not len(old_categories):
+            errors.append(f'No assigned categories for {url}')
+            continue
+
+        category = CFGOVPageCategory(name=old_categories[0].name)
+
+        tags = page.tags.all()
+        sidefoot = get_stream_data(page, 'sidefoot')
+        sidefoot_fields = []
+        continue_outer_loop = False
```

Comment:
It's used to skip actually deleting/replacing the page if an error is encountered in the nested `for` loop below. Your earlier comment about having this method only operate on a single page is prescient here, because in this case it would just be a short-circuit return instead of needing to do the variable hackery to continue the outer loop